### PR TITLE
Skeletized tooltip help component

### PR DESCRIPTION
**Pull request by louiscoste**
Added skeleton mixin to `tooltip-help`

### CODE REVIEW

**Review by dbatiste — COMMENTED** [review]
This looks pretty good. I'll just take it for a test-drive before approving.

**Comment by dbatiste** [comment]
We should include a visual-diff test that captures the skeleton mode.

**Comment by dbatiste** [comment]
Looks like we get some of the underline peeking out from under the skeleton. We probably just need to apply `text-decoration: none;` when in skeleton mode.
image

File: components/tooltip/tooltip-help.js [hunk 1]
```diff
@@ -103,12 +107,14 @@ class TooltipHelp extends FocusMixin(FocusVisiblePolyfillMixin(LitElement)) {
 			'd2l-body-small': !this.inheritFontStyle
 		};
 		return html`
-			<button id="d2l-tooltip-help-text" class="${classMap(classes)}" type="button">
+			<button id="d2l-tooltip-help-text" class="${classMap(classes)} d2l-skeletize" type="button">
```
Comment (dlockhart) [inline components/tooltip/tooltip-help.js:110]: Suggest adding this to the `classes` so there isn't a leading space when `d2l-body-small` isn't there:

```javascript
const classes = {
	'd2l-body-small': !this.inheritFontStyle,
	'd2k-skeletize': true
};
```

**Review by dlockhart — APPROVED** [review]
LGTM!

**Review by dbatiste — APPROVED** [review]

File: components/tooltip/tooltip-help.js [hunk 2]
```diff
@@ -76,6 +77,9 @@ class TooltipHelp extends FocusMixin(FocusVisiblePolyfillMixin(LitElement)) {
 			d2l-tooltip {
 				cursor: text;
 			}
+			:host([skeleton]) button.d2l-skeletize {
```
Comment (dbatiste) [inline components/tooltip/tooltip-help.js:80]: If we use `:host([skeleton]) #d2l-tooltip-help-text.d2l-skeletize` selector, we can avoid the `!important`. We try to only rely on `!important` when absolutely necessary.

**Review by dbatiste — APPROVED** [review]
Perfect... thanks Louis!

**Pull request merged**